src/soroban/assembled_transaction.ts

```diff
@@ -54,8 +53,14 @@ export type MethodOptions = {
    * The fee to pay for the transaction. Default: BASE_FEE
    */
   fee?: number;
+  /**
+   * The maximum amount of time to wait for the transaction to complete. Default: {@link DEFAULT_TIMEOUT}
+   */
+  timeoutInSeconds?: number;
```

Comment:
can we make this required rather than have a default?

Reply:
I would way rather have a default! That way you don't need to include a second options object on every call. If we make it required, every function call from the generated Contract Clients would look like this:

```ts
await myContract.myMethod(
  { args: 'for', my: 'method', ... },
  { timeoutInSeconds: 10 }
)
```

Do you think this is important enough to bring to people's attention that we should uglify every method call in this way, or do you think there's a good-enough default?